packages/eslint-config-olo/rules/react.js

```diff
@@ -67,22 +75,29 @@ module.exports = {
     }],
     // Prevent React to be incorrectly marked as unused
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md
-    'react/jsx-uses-react': [2, { 'pragma': 'React' }],
+    'react/jsx-uses-react': 2,
     // Prevent variables used in JSX to be incorrectly marked as unused
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-vars.md
     'react/jsx-uses-vars': 2,
+    // Prevent missing parentheses around multilines JSX
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/wrap-multilines.md
+    'react/jsx-wrap-multilines': [2, {
+      declaration: 'parens',
+      assignment: 'parens',
+      return: 'parens'
+    }],
     // Prevent usage of dangerous JSX properties
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-danger.md
     'react/no-danger': 0,
     // Prevent usage of deprecated methods
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-deprecated.md
-    'react/no-deprecated': [1, { 'react': '0.14.0' }],
+    'react/no-deprecated': 1,
     // Prevent usage of setState in componentDidMount
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-mount-set-state.md
-    'react/no-did-mount-set-state': [2, 'allow-in-func'],
+    'react/no-did-mount-set-state': 2,
```

Comment:
I may be missing some context, this seems like a change that is more restrictive? It previously allowed callback functions... https://github.com/yannickcr/eslint-plugin-react/commit/4c167ebc16c1593489b61886716c48cfd2044f2d#diff-7363406d1b2e67feb3b50cbde0fff1caR60

Reply:
The rule was changed to the inverse -- `disallow-in-func` -- so the removal of that configuration property is equivalent with the new version of eslint.